docs/pages/installation/gke.md

```diff
@@ -5,19 +5,31 @@ permalink: installation/gke
 ---
 
 # Quick Start with Google Kubernetes Engine (GKE)
-You may perform the steps outlined under [Managed Kubernetes](#managedk8s) following by hand or run the [generate_kubeconfig_gke.sh](./generate_kubeconfig_gke.sh) shell script using the desired ServiceAccount name and Namespace arguments, like so:
 
-`./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
+Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
+<br>Utilize the token to run the following command:
+
+| command           | flag                | function                                                     | Usage                     |
+|:------------------|:-------------------:|:-------------------------------------------------------------|:--------------------------|
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(GKE). | `mesheryctl system config gke --token "PATH TO TOKEN"` |
+
+Once configured, head over to the Quick Start Guide and continue with the steps outlined for <a href = "https://github.com/layer5io/meshery/blob/d75b6eec23021a1d8d3a38ec890fb10d44af3a35/docs/pages/installation/gke.md#manual-configuration">GKE</a>.
+
+## **Manual Configuration**
 
-Having run this script, supply the generated file `config-cluster-admin-sa-gke-default.yaml` in your Meshery settings page.
+Follow the below mentioned steps to set up manually:
+
+You may perform the steps outlined under [Managed Kubernetes](#managedk8s), following by hand or run the [generate_kubeconfig_gke.sh](./generate_kubeconfig_gke.sh) shell script using the desired ServiceAccount name and Namespace arguments, like so:
+
+`./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
 
-## Configuration
-This is where you configure your settings on the adaptor(Istio etc) and other things 
+Having run this script, supply the generated file config-cluster-admin-sa-gke-default.yaml in your Meshery settings page.
```

Comment:
Can this be made a little more formal & detailed? Like : 
- After running this script, start the meshery server & navigate to `localhost:3000/settings` and provide the generated file xyz under Kube-config.

Reply:
Good suggestion. Acknowledging that most Meshery users will have either deployed Meshery locally at `localhost:9081` or within a Kubernetes cluster and exposed access to it (which could be any IP address and port), perhaps, we should be unspecific in our reference to Meshery's location (URL).

Reply:
@chandrashritii a reminder that the hostname and/or port number may change from deployment to deployment.